lib/oli/authoring/locks.ex

```diff
@@ -159,27 +169,27 @@ defmodule Oli.Authoring.Locks do
     from(pr in PublishedResource,
       where: pr.publication_id == ^publication_id and not is_nil(pr.locked_by_id),
       select: pr)
-    |> Repo.update_all(set: [locked_by_id: nil, lock_updated_at: nil])
+    |> Repo.update_all(set: [locked_by_id: nil])
```

Comment:
I'm not sure if its useful to the system, but it would seem more fitting to me to actually set the `lock_updated_at` here to either the current datetime (since technically setting to null is updating it) or setting it to null if it really is not useful information. Retaining the last `lock_updated_at` value though seems a bit inconsistent to me